plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OraclePoolConnectionFactory.java

```diff
@@ -59,6 +60,7 @@ public OraclePoolConnectionFactory(
         dataSource.setValidateConnectionOnBorrow(true);
         dataSource.setConnectionProperties(connectionProperties);
         dataSource.setInactiveConnectionTimeout(toIntExact(inactiveConnectionTimeout.roundTo(SECONDS)));
+        dataSource.setConnectionWaitTimeout(toIntExact(connectionWaitTimeout.roundTo(SECONDS)));
```

Comment:
`setConnectionWaitTimeout` is deprecated. Can we use `setConnectionWaitDuration` instead?

Reply:
Done!